src/main/kotlin/com/kotlin/boardproject/domain/user/domain/User.kt

```diff
@@ -1,18 +1,21 @@
-package com.kotlin.boardproject.model
+package com.kotlin.boardproject.domain.user.domain
 
-import com.kotlin.boardproject.auth.ProviderType
-import com.kotlin.boardproject.common.enums.CurrentStatus
-import com.kotlin.boardproject.common.enums.Role
-import org.hibernate.annotations.GenericGenerator
+import com.kotlin.boardproject.domain.post.domain.LikePost
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end